### PR TITLE
Timeseries search sdk 2.0

### PR DESCRIPTION
**Pull request by follan**
`RootAssetSelect` is disabled since `assetSubtree` is not supported in v1 of API yet

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-337.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.test.tsx [hunk 1]
```diff
@@ -56,49 +67,62 @@ describe('TimeseriesSearch', () => {
...
+      search: { query: 'value' },
       limit: 100,
-      assetSubtrees: undefined,
+      filter: { assetSubtrees: undefined },
     });
   });
 
-  it('should update assetId with user-selected root asset id', done => {
+  // assetSubtree query is not supported yet. Check with f1
+  xit('should update assetId with user-selected root asset id', done => {
```
Comment (xitman78) [inline src/components/TimeseriesSearch/TimeseriesSearch.test.tsx]: Why is this test skipped with 'xit'?

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 2]
```diff
@@ -128,10 +134,17 @@ export class TimeseriesSearch extends React.Component<
   }
 
   async componentDidMount() {
+    if (!this.context) {
+      console.error(
+        'Client SDK Context has not been provided. Use ClientSDKProvider to wrap the component.'
```
Comment (xitman78) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Please use text  constants from `src/constants/errorMessages.ts`

File: src/components/TimeseriesSearch/stories/TimeseriesSearch.stories.tsx [hunk 3]
```diff
@@ -82,9 +125,11 @@ storiesOf('TimeseriesSearch', module).add(
     setupMocks();
 
     return (
-      <TimeseriesSearch
-        onTimeserieSelectionChange={onTimeserieSelectionChange}
-      />
+      <ClientSDKProvider client={fakeClient}>
```
Comment (xitman78) [inline src/components/TimeseriesSearch/stories/TimeseriesSearch.stories.tsx]: It's better to use decorators to provide the client in stories.

**Pull request merged**